installation/production_run.md

```diff
@@ -201,9 +223,7 @@ Reference the following files for detailed configuration:
 
 2. **API URL Configuration**
 
-   - Frontend needs two API URLs configured:
-     - `MARBLE_API_URL_CLIENT`: URL for browser requests (public URL)
```

Comment:
I am preparing the path the when the whole `/config` PR will be merged. At that point, technically, the frontend will only require:

 - ENV
 - NODE_ENV
 - MARBLE_API_URL_SERVER
 - SESSION_SECRET
 - _+ anything that is secret_